src/Resolver/TemplatePathStack.php

```diff
@@ -236,13 +246,15 @@ public function addPath($path)
      */
     public function clearPaths()
     {
-        $this->paths = new SplStack();
+        /** @psalm-var PathStack $paths */
```

Comment:
I don't understand what is going on here :thinking:

Reply:
I introduced a psalm-type on this class as `PathStack = SplStack<string>` - is that the wrong way to do it? The reason for the local variable here is so that I can tell psalm what it is before assigning to `$this->paths` otherwise I get a psalm error. Can you tell me a better way?

Reply:
Ooooh, now I see it! It's all good, I was just confused about it, thanks for clarifying!